pkgs/development/libraries/speechd/default.nix

```diff
@@ -28,18 +39,43 @@ let
       throw "You need to enable at least one output module.";
 in stdenv.mkDerivation rec {
   pname = "speech-dispatcher";
-  version = "0.8.8";
+  version = "0.9.1";
 
   src = fetchurl {
-    url = "http://www.freebsoft.org/pub/projects/speechd/${pname}-${version}.tar.gz";
-    sha256 = "1wvck00w9ixildaq6hlhnf6wa576y02ac96lp6932h3k1n08jaiw";
+    url = "https://github.com/brailcom/speechd/releases/download/${version}/${pname}-${version}.tar.gz";
```

Comment:
Why are the source tarball chosen over a `fetchFromGitHub`-based one? The latter seems easier to update automatically.